modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/action/RequestType.java

```diff
@@ -15,32 +15,56 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.replicator.listener;
+package org.apache.ignite.internal.table.distributed.replicator.action;
 
 /**
- * The listener response returns when the replication process completes.
+ * Transaction operation type.
  */
-public class ListenerInstantResponse implements ListenerResponse {
-    /**
-     * Result.
-     */
-    private Object result;
-
-    /**
-     * The constructor.
-     *
-     * @param result Result.
-     */
-    public ListenerInstantResponse(Object result) {
-        this.result = result;
-    }
-
-    /**
-     * Gets a result.
-     *
-     * @return Some result of the replication process.
-     */
-    public Object result() {
-        return result;
-    }
+public enum RequestType {
+    /** RW get operation. */
+    RW_GET,
+
+    /** RW get all operation. */
+    RW_GET_ALL,
+
+    /** RW delete operation. */
+    RW_DELETE,
+
+    /** RW delete all operation. */
```

Comment:
Let's be consistent with javadocs and remove them, cause instance names are themselves pretty clear.

Reply:
Staied all javadoc empty.